src/styles/utils/_animations.scss

```diff
@@ -0,0 +1,19 @@
+@keyframes seconds-arrow-move {
+  from {
+    transform: translateX(-50%) rotate(getSecondsStartPos($stopwatch-time));
+  }
+
+  to {
+    transform: translateX(-50%) rotate(0deg);
+  }
+}
+
+@keyframes minutes-arrow-move {
+  from {
+    transform: translateX(-50%) rotate(getMinutesStartPos($stopwatch-time));
+  }
+
+  to {
+    transform: translateX(-50%) rotate(0deg);
+  }
+}
```

Comment:
You don't need to use two keyframes for same animations.

Reply:
It is the same animation, there are different functions in start rotate position

Reply:
agree with @MaksymMaliuk

Reply:
u can use css steps